src/data/fintype/card.lean

```diff
@@ -155,6 +155,15 @@ begin
     exact ⟨⟨k, hk⟩, mem_univ _, rfl⟩ }
 end
 
+@[to_additive]
+lemma finset.univ_prod_eq_range_prod [comm_monoid β] {n : ℕ} {c : fin n → β} :
```

Comment:
```suggestion
lemma finset.univ_prod_eq_range_prod [comm_monoid β] {n : ℕ} (c : fin n → β) :
```
There is no harm in making `c` explicit here. You won't need to write it when using `rw` or `apply`, and it can be useful to have it explicit when you want to `have := finset.univ_prod_eq_range_prod c`.